gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -51,6 +60,7 @@
 import dagger.hilt.android.AndroidEntryPoint;
 import java8.util.Optional;
 import javax.inject.Inject;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Don't see this used here, so probably can remove it. Also, \@NotNull is our default so shouldn't be required.